src/kbe/pbe.py

```diff
@@ -351,7 +353,9 @@ def print_ini(self):
 
     def ewald_sum(self, kpts=None):
         dm_ = self.mf.make_rdm1()
-        nk, nao, nao = dm_.shape
+        nk, nao, nao_2nd_dim = dm_.shape
+        assert nao == nao_2nd_dim
+        del nao_2nd_dim
```

Comment:
For this line (unlike 233-235), I think it is guaranteed that `nao` = `nao_2nd_dim`.
```suggestion
        nk, nao, _ = dm_.shape
```
I'm fine either way.

Reply:
in this case I can also abbreviate:
```python3
 nk, nao, _ = dm_.shape
```
or
```python3
 nk, nao = dm_.shape[:2]
```

Reply:
your call! :)